rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -730,6 +730,15 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 		ui->fsrSharpeningStrength->setValue(fsr_sharpening_strength_def);
 	});
 
+	//DoubleSpinBoxes
+	m_emu_settings->EnhanceDoubleSpinBox(ui->TextureLODBias, emu_settings_type::TextureLodBias);
+	SubscribeTooltip(ui->TextureLODBias, tooltips.settings.texture_lod_bias);
+	ui->TextureLODBias->setRange(-16, 16);
```

Comment:
Why this limitation? If it allows for higher values, then either those values are a valid choice or nonsense and should be reduced instead

Reply:
From my understanding of
https://github.com/RPCS3/rpcs3/pull/12866#discussion_r1002588254
If a game sets -16 and user sets -32 = -48 then from what I can tell that will exceed what is possible.
Setting this limit prevents users that don't know what the game sets from using this setting incorrectly. The setting isn't nonsense but I wanted to prevent bad math.
Good example of the setting being used correctly is game sets +16 user sets -32 = -16. But how does the user know what the game set?

Reply:
Simply set it to +32 -32 is fine.

Reply:
It clamps the value to a valid range anyway, otherwise it would make the backends be pissed.

Reply:
Thanks, I changed it. Good to know too